app/cells/folio/console/ui/alert/alert.js

```diff
@@ -78,8 +78,31 @@ window.FolioConsole.Ui.Alert.create = (data) => {
 }
 
 window.Folio.Stimulus.register('f-c-ui-alert', class extends window.Stimulus.Controller {
+  static values = { autohide: Boolean }
+
+  connect() {
+    if (this.autohideValue) {
+      this.autohideTimeout = setTimeout(() => {
+        const closeBtn = this.element.querySelector('.f-c-ui-alert__close')
+        if (closeBtn) closeBtn.click()
+      }, 5000)
+    }
+  }
+
+  disconnect() {
+    this.clearAutohideTimeout()
+  }
+
   close (e) {
     e.preventDefault()
+    this.clearAutohideTimeout()
     this.element.parentNode.removeChild(this.element)
   }
+
+  clearAutohideTimeout() {
+    if (this.autohideTimeout) {
+      clearTimeout(this.autohideTimeout)
+      this.autohideTimeout = null
+    }
+  }
 })
```

Comment:
v `window.FolioConsole.Ui.Alert.create` se pracuje s `data.autohide`. Prehod to prosim rovnou na to nove reseni, ze se jen nastavi value. At tam neni timeout logika dvakrat.